src/main/java/org/jboss/tools/intellij/openshift/ui/service/CreateServiceDialog.java

```diff
@@ -45,14 +46,26 @@ public void setServiceTemplates(ServiceTemplate[] serviceTemplates) {
         serviceTemplatesComboBox.setRenderer(new DefaultListCellRenderer() {
             @Override
             public Component getListCellRendererComponent(JList<?> list, Object value, int index, boolean isSelected, boolean cellHasFocus) {
-                value = ((ServiceTemplate) value).getName() + " " + ((ServiceTemplate) value).getPlan();
+                value = ((ServiceTemplate) value).getName();
                 return super.getListCellRendererComponent(list, value, index, isSelected, cellHasFocus);
             }
         });
+        serviceTemplatesComboBox.addItemListener(item -> templateSelected((ServiceTemplate) item.getItem()));
         serviceTemplatesComboBox.setModel(new DefaultComboBoxModel(serviceTemplates));
+        serviceTemplatesComboBox.setSelectedIndex(-1);
```

Comment:
I'm wondering why there is this extra set selection ?

Reply:
That's because the first element is already the selected one so setSelectedIndex(0) did not fire the listener to be called. But I replace by a more understandable code section